api/api.py

```diff
@@ -227,7 +227,9 @@ def k8s_resource(workload: str, new_name: str = "",
                  port_forwards: Union[str, int, List[int]] = [],
                  extra_pod_selectors: Union[Dict[str, str], List[Dict[str, str]]] = [],
                  trigger_mode: TriggerMode = TRIGGER_MODE_AUTO, resource_deps: List[str] = []) -> None:
-  """Configures a kubernetes resources
+  """Configures the Kubernetes resource of the given name. Note: Tilt does an amount of resource configuration
```

Comment:
I don't think this comment is accurate. Most people use k8s_reosurce for port-forwards, which Tilt does not do for you

Reply:
I might split this into two comments: one here that just something like "Resource names are calculated automatically, as described in [link]", and then add a comment to "extra_pod_selectors" (or whatever parameter the user was confused about) with something like "You should use this parameter IF [you're in edge case X, Y, or Z]"

Reply:
> Configures the Kubernetes resource of the given name. Tilt assembles (and names) Kubernetes resources automatically, as described in `Tiltfile Concepts: Resources <tiltfile_concepts.html#resources>`_). Calling ``k8s_resource`` is *optional*; you can use this function to configure port forwarding for your resource, to rename it, or to adjust any of the other settings specified below.

Better?

Reply:
did you intend to push up an update?

Reply:
i was gonna push after you OK'd the copy :)